pkg/apis/management.cattle.io/v3/authz_types.go

```diff
@@ -171,6 +171,12 @@ type GlobalRole struct {
 	// cluster besides the local cluster. To grant permissions in the local cluster, use the Rules field.
 	// +optional
 	InheritedClusterRoles []string `json:"inheritedClusterRoles,omitempty"`
+
+	// NamespacedRules are the rules that are active in each namespace of this GlobalRole.
+	// * has no special meaning in the keys - these keys are read as raw strings
+	// and must exactly match with one existing namespace.
+	// +optional
+	NamespacedRules map[string][]rbacv1.PolicyRule `json:"namespacedRules,omitempty"`
```

Comment:
Nit: Should we hint that this field is for the local cluster only or is this _**easily**_ inferred from `InheritedClusterRoles` being "other clusters" and `Rules` being local only?

Reply:
Good point, I think it's worth explicitly stating.

Reply:
Added a line to clarify